girder/cumulus/server/volume.py

```diff
@@ -33,6 +33,7 @@
 from cumulus.constants import VolumeType
 from cumulus.constants import VolumeState
 from cumulus.common.girder import get_task_token, _get_profile
+from cumulus.aws.ec2 import get_ec2_client
 
```

Comment:
No. I had a mock decorator for the get_ec2_client that was pointing the volume rest object rather than the aws model. Once I fixed that I was able to remove this unneeded import.